src/Test/Utilities/DisposableFile.cs

```diff
@@ -51,5 +53,31 @@ public void Dispose()
                 }
             }
         }
+
+        [DllImport("kernel32.dll", PreserveSig = false)]
+        private static extern void SetFileInformationByHandle(SafeFileHandle handle, int fileInformationClass, ref uint fileDispositionInfoDeleteFile, int bufferSize);
+
+        private const int FileDispositionInfo = 4;
+
+        internal static void PrepareDeleteOnCloseStreamForDisposal(FileStream stream)
+        {
+            // tomat: Set disposition to "delete" on the stream, so to avoid ForeFront EndPoint
+            // Protection driver scanning the file. Note that after calling this on a file that's open with DeleteOnClose, 
+            // the file can't be opened again, not even by the same process.
+            uint trueValue = 1;
+            SetFileInformationByHandle(stream.SafeFileHandle, FileDispositionInfo, ref trueValue, sizeof(uint));
```

Comment:
Side note, this is not going to work when running in an AppContainer process and the file came from the broker (ie you were passed a file from the user's user documents or pictures, or something that appcontainer processes don't automatically have access to). SafeFileHandle in this case will return an invalid handle: https://github.com/dotnet/corefx/blob/4285afc7a857d01b889c8449be683af2f2508266/src/System.IO.FileSystem/src/System/IO/WinRTFileStream.cs#L42.